src/platform/PICO/include/platform/platform.h

```diff
@@ -131,3 +131,16 @@ extern uint32_t systemUniqueId[3];
 
 #define USE_LATE_TASK_STATISTICS
 
+#ifndef DEFAULT_VOLTAGE_METER_SCALE
+// 100 = 1.00x (100%) scaling; override per target/board to match its VBAT divider
+#define DEFAULT_VOLTAGE_METER_SCALE   100
+#endif
+
+// TODO: identify correct values for RP2350 internal temp sensor, or move conversion into adc_pico.c.
+#if defined(USE_ADC_INTERNAL)
+#error "PICO: Internal temp/Vref calibration constants are not verified for RP2350. Provide correct values or implement conversion in adc_pico.c."
```

Comment:
Maybe use just warning, it will trigger -Werror in normal build anyway
```suggestion
#warning "PICO: Internal temp/Vref calibration constants are not verified for RP2350. Provide correct values or implement conversion in adc_pico.c."
```